### ISSUE TITLE
Dual fuel HP w/ switchover temperature

### ISSUE THREAD

**Issue by shorowit** [NREL/OpenStudio-HPXML#1265]
Currently we disable the gas furnace above the switchover temperature. Should we instead allow it to continue to operate as backup (assuming E+ allows it)? Otherwise you can get large unmet loads if using a low switchover temperature.

cc @jonwinkler @yzhou601

**Comment by benjamintaube** [comment]
E+ allows this through Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation and Minimum Outdoor Dry-Bulb Temperature for Compressor Operation fields (part of the unitary equipment and/or coils depending on the type of equipment). So long as [max OA DBT for supplemental heating] > [min OA DBT for compressor operation], the gas coil should still run above the switchover temperature (no longer switchover though...just HP shutoff temp). 

How low of a switchover temperature is allowed? Depending on the performance curves of the ASHP, it might make sense to ensure that the switchover temperature isn't so low that the compressor operates at a COP < 1. With the strategy above, if [min OA DBT for compressor operation] is very low, the compressor could still be running at COP < 1 because the supplemental coil comes after the DX coil.

**Comment by jonwinkler** [comment]
@benjamintaube, Yes, that's our understanding as well. However, with residential dual fuel systems, the gas furnace and heat pump cannot operate simultaneously. So we essentially want the heat pump to turn off above the min OA DBT for compressor operation when the heat pump can't meet the load.

**Comment by benjamintaube** [comment]
@jonwinkler got it. In @shorowit original question, it sounded like he was considering allowing the gas furnace to continue operating above the min OA DBT for compressor operation _alongside_ the compressor just for cases with particularly low switchover temps, as described in [this conference paper](https://www.osti.gov/servlets/purl/1885313) from ORNL.

**Comment by shorowit** [comment]
Superceded by https://github.com/NREL/OpenStudio-HPXML/issues/1270
